styles/widgets/generic/scheduler.generic.less

```diff
@@ -370,7 +370,7 @@
 
 .dx-recurrence-editor-container {
     position: relative;
-    margin-top: 15px;
+    margin-left: 84px;
```

Comment:
Check recurrence editor visibility in other themes and rtl-mode

Reply:
OK. In the next PR